sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -157,7 +157,7 @@
      *
      * <p>Add a setting with the key "prodDBConnection" and value "db_connection".</p>
      *
-     * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.setSettingWithResponse#ConfigurationSetting}
+     * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.setSetting#string-string}
```

Comment:
You changed this.. but did you also change the corresponding sample .java tag?

Reply:
I think it used a wrong codenippet sample. If you ctrl+shift+f to find "com.azure.data.appconfiguration.configurationasyncclient.setSetting#string-string" there is no one use this code sample. So I changed to current version.